File: 1758-minimum-changes-to-make-alternating-binary-string/1758-minimum-changes-to-make-alternating-binary-string.js
/**
 * @param {string} s
 * @return {number}
 */
function minOperations(s) {
    let count = 0;
    for(let i = 0; i < s.length; i++){
        if(s[i] !== (i % 2).toString()){
            count++;
        }
    }
    return Math.min(count,s.length - count);
    
};

File: 1758-minimum-changes-to-make-alternating-binary-string/1758-minimum-changes-to-make-alternating-binary-string.java
class Solution {
    public int minOperations(String s) {
        
        int count = 0;
        for(int i = 0; i < s.length(); i++){
             if (s.charAt(i) != Character.forDigit(i % 2, 10)) count++;
        }
        return Math.min(count,s.length() - count);
}
};